hvplot/util.py

```diff
@@ -290,14 +290,24 @@ def process_crs(crs):
         crs = crs.to_wkt()
 
     errors = []
-    if isinstance(crs, (str, int)):  # epsg codes
+    if isinstance(crs, (str, int, pyproj.Proj)):
+        wkt = crs
+        if isinstance(crs, (str, int)):  # epsg codes
+            try:
+                wkt = pyproj.CRS.from_epsg(crs).to_wkt()
+            except Exception as e:
+                errors.append(e)
         try:
-            crs = pyproj.CRS.from_epsg(crs).to_wkt()
+            return proj_to_cartopy(wkt)  # should be all proj4 or wkt strings
         except Exception as e:
             errors.append(e)
-    if isinstance(crs, (str, pyproj.Proj)):  # proj4/wkt strings
+
+    if isinstance(crs, (str, int)):
```

Comment:
This idea is that this is purely additive. It should have no impact on code paths that were already working.